diff --git a/node_modules/grommet/components/Meter/index.d.ts b/node_modules/grommet/components/Meter/index.d.ts
index fea8c09..762ed4a 100644
--- a/node_modules/grommet/components/Meter/index.d.ts
+++ b/node_modules/grommet/components/Meter/index.d.ts
@@ -19,7 +19,7 @@ export interface MeterProps {
   round?: boolean;
   size?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'full' | string;
   thickness?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | string;
-  type?: 'bar' | 'circle' | 'pie';
+  type?: 'bar' | 'circle' | 'pie' | 'semicircle';
   value?: number;
   values?: {
     color?: string;
